srcpkgs/NetworkManager-fortisslvpn/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'NetworkManager-fortisslvpn'
+pkgname=NetworkManager-fortisslvpn
+version=1.3.90
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static"
+hostmakedepends="automake gettext-devel glib-devel intltool libtool pkg-config"
+makedepends="ppp-devel gtk+3-devel libglib-devel libnma-devel libsecret-devel"
+depends="openfortivpn"
+short_desc="NetworkManager VPN plugin for OpenFortiVPN"
+maintainer="Twilighttony <starrbeartony14@zoho.com>"
+license="GPL-2.0-or-later"
+homepage="https://wiki.gnome.org/Projects/NetworkManager"
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=884a7432bb38ae9c942e094f23d0a3feb504432153c3d2869ab71a4dd244d0fb
+system_accounts="_nm_openfortivpn"
+lib32disabled=yes
+_nm_openfortivpn_homedir="/var/empty"
+_pppver=2.4.9
+
+configure_args="--libexecdir=/usr/lib/networkmanager --with-pppd-plugin-dir=/usr/lib/pppd/$_pppver"
```

Comment:
You should probably move this to the "configure_args" line above.

Reply:
I have fixed this now.